ota-plus-web/app/reactapp/src/pages/ImpactAnalysis.jsx

```diff
@@ -19,7 +19,6 @@ class ImpactAnalysis extends Component {
     const { stores } = this.props;
     const { softwareStore, impactAnalysisStore } = stores;
```

Comment:
impactAnalysisStore is not needed anymore

Reply:
Removed.